jssource/src_files/include/javascript/sugar_3.js

```diff
@@ -2028,11 +2028,13 @@ sugarListView.get_num_selected_string = function () {
  * @returns {number}
  */
 sugarListView.get_num_selected = function () {
-  var the_form = document.MassUpdate;
-  if (typeof the_form != 'undefined' && the_form.select_entire_list.value == 1) {
-    var selectCount = $("input[name='selectCount[]']:first");
-    if (selectCount.length > 0)
-      return parseInt(selectCount.val().replace("+", ""));
+  if (document.MassUpdate != 'undefined') {
```

Comment:
Hm, in which case should "document.MassUpdate" be 'undefined'? Isn't this just skipping get_checks_count in call cases?

Reply:
In my scenario, (7.8.x instance upgraded to 7.10.x) once I click on a listview item I get the same error in the detail view (hundreds of times)

Reply:
Based on when the error occurs, it should be defined when on a page where you can actually use the mass update functionality, but then when you switch to another page via ajax that doesn't have that, it becomes undefined. I'm not entirely sure why the code stays around at all, but 🤷‍♂ I don't think it should break anything, and if you want to test it I'd be happy to have the functionality validated :)

Reply:
You are never checking if it is undefined, you are comparing it against a string "undefined".

Reply:
🤦‍♂ oops, I've updated it to compare with the type now.